packages/cms-data-sync/src/news/news.data-migration.ts

```diff
@@ -1,113 +1,89 @@
-/* eslint-disable no-await-in-loop */
-/* eslint-disable no-restricted-syntax */
 /* eslint-disable @typescript-eslint/no-non-null-assertion */
 
-import { Entry, SysLink } from 'contentful-management';
 import { Document } from '@contentful/rich-text-types';
-
 import { newsQuery } from './news.queries';
 import {
   FetchNewsQuery,
   FetchNewsQueryVariables,
 } from '../autogenerated-gql/graphql';
-
 import {
-  clearContentfulEntries,
   convertHtmlToContentfulFormat,
   createAsset,
   getSquidexAndContentfulClients,
   logger,
-  publishContentfulEntries,
 } from '../utils';
+import { migrateFromSquidexToContentfulFactory } from '../utils/migration';
```

Comment:
maybe it would be nice to add 
```
export * from './migration';
```
to `packages/cms-data-sync/src/utils/index.ts` so we can import this from `'../utils'` only

Reply:
so i'm not a fan having those index files anywhere, we only had them in packages because they were necessary - we had to point package.json to a single file where all the exports were, but for internal use i think its a little misleading. TypeScript actually helps you with auto completing these paths so it's not like you have to type them every time and I see no other benefit of having those re-exporting index files